web/src/app/layout.tsx

```diff
@@ -15,16 +16,19 @@ export const metadata: Metadata = {
 export default function RootLayout({children,}: Readonly<{children: React.ReactNode;}>) {
   return (
     <html lang="en">
-      <head>
-        <ColorSchemeScript defaultColorScheme="dark"/>
-      </head>
+		<head>
+			<ColorSchemeScript defaultColorScheme="dark"/>
```

Comment:
is this a whitespace mistake? or just a github issue

Reply:
github issue